static/src/javascripts/projects/common/modules/commercial/track-ad.js

```diff
@@ -0,0 +1,30 @@
+define([
+    'Promise',
+    'common/utils/mediator'
+], function(Promise, mediator) {
+    return trackAd;
+
+    function trackAd(id) {
+        return new Promise(function (resolve, reject) {
+            var onAdLoaded = function (event) {
+                if (event.slot.getSlotElementId() === id) {
+                    unlisten();
+                    resolve(!event.isEmpty);
```

Comment:
I don't have a use case for it, I just wasn't sure if I'd expect a boolean as part of the interface. I suppose it's fine.